tests/user_commands/tests.py

```diff
@@ -174,6 +174,15 @@ def test_check_migrations(self):
                 self.assertTrue(check_migrations.called)
         finally:
             dance.Command.requires_migrations_checks = requires_migrations_checks
+    
+    def test_call_command_unrecognized_options(self):
+        """
+        When passing unrecognized options to call command, an command error
+        should be raised (#27787).
```

Comment:
A ticket reference isn't needed -- that's only for obscure issues where the behavior isn't straightforward.